script/test

```diff
@@ -46,7 +46,9 @@ readonly THREAD_VERSION="${THREAD_VERSION:-1.2}"
 readonly INTER_OP="${INTER_OP:-0}"
 readonly VERBOSE="${VERBOSE:-0}"
 readonly BORDER_ROUTING="${BORDER_ROUTING:-1}"
+readonly BORDER_ROUTING_NAT64="${BORDER_ROUTING_NAT64:-1}"
 readonly INTER_OP_BBR="${INTER_OP_BBR:-1}"
+readonly NAT64="${NAT64:-0}"
```

Comment:
BORDER_ROUTING_NAT64 is a switch on OT to enable NAT64 prefix advertisement.

NAT64 is a switch on OTBR to enable Tayga.
Reverted the change of NAT64 here since it's set as 0 for all tests at this moment.